packages/http/libraries/core/src/http/adapter/InversifyHttpAdapter.ts

```diff
@@ -279,6 +296,20 @@ export abstract class InversifyHttpAdapter<
     return requestHandlerList;
   }
 
+  #printController(
+    controllerName: string,
+    path: string,
+    controllerMethodMetadataList: ControllerMethodMetadata[],
+  ): void {
+    console.log(`${controllerName} {${path}}:`);
```

Comment:
I know I said yes to this approach, but let's make an abstraction with some logging levels. I think [winston](https://github.com/winstonjs/winston?tab=readme-ov-file#logging-levels) does a pretty good job in this regard.

What do you think about it?

Reply:
I agree, but I consider that a logging system can be large and critical enough to be out of scope in this PR.